File: Homework 1/turker_ege.php
<?php

$playlists = array( "pop" => 104, "rock" => 243, "jazz" => 72, "electronic" => 24); 

echo $playlists["pop"];

$playlists["hiphop"] = 44;

unset($playlists["rock"]);

$playlists["jazz"] = 83;

if (array_key_exists("electronic",$playlists)){
echo "\nKey exists";
}
else{
echo "\nKey doesn't exist";
}
if (array_key_exists("rock",$playlists)){
echo "\nKey exists";
}
else{
echo "\nKey doesn't exist";
}

if (in_array(104, $playlists)){
echo "\nValue exists";
}
else{
echo "\nValue doesn't exist";
}
if (in_array(243,$playlists)){
echo "\nValue exists";
}
else{
echo "\nValue doesn't exist";
}

foreach($playlists as $key => $value) {
    foo($key, $value);
}

function foo($key, $value) {
  echo "\n" . $key . ": " . $value ;
} 

?>

File: Homework 1/turker_ege.py
def foo(key, value):
  print(key, value) 

playlists = { 'pop': 104,'rock': 243, 'jazz': 72, 'electronic': 24  }

print(playlists['pop'])

playlists['hiphop'] = 44

del playlists['rock']

playlists['jazz'] = 83

if 'electronic' in playlists:
    print('Key exists !')
else:
    print('Key doesnt exist !')
    
if 'rock' in playlists:
    print('Key exists !')
else:
    print('Key doesnt exist !')

if 104 in playlists.values():
    print('Value exists !')
else:
    print('Value doesnt exist !')
if 243 in playlists.values():
    print('Value exists !')
else:
    print('Value doesnt exist !')
    
for key in playlists:
    value = playlists[key]
    foo(key,value)
    

